features/facebookImport/src/components/importExplanationExpandable/importExplanationExpandable.jsx

```diff
@@ -200,11 +200,7 @@ const ImportExplanationExpandable = ({
             {Object.values(importSections).map((section, index) => (
                 <div key={index} className={`section ${section}`}>
                     <div
-                        onClick={
-                            section === "explore"
-                                ? () => {}
-                                : () => onUpdateImportStatus(section)
-                        }
+                        onClick={() => onUpdateImportStatus(section)}
```

Comment:
I suppose this is the actual bug fix? Where people would sometimes not be in _explore_ after already having selected a file, or maybe even the thing where people would sometimes end up on the _import_ view instead of _overview_ if they already have an existing file? (Noticed both issues a few times myself, but cannot reproduce them yet)

Reply:
This actually is reverting the change that you cannot go to explore if you havent uploaded a file, which Arne told me they don't want as a behaviour